docs/setting-up-a-validator.md

```diff
@@ -175,12 +207,53 @@ journalctl -u orchestrator.service -f --output cat
 journalctl -u geth.service -f --output cat
 ```
 
-## Wait for it
+## Observe Sync Status and Time Remaining
+You will need to wait for your Gravity Bridge node and Ethereum Node to fully sync before progressing in the instructions </br>
+### Ethereum
+You can view the status of your Ethereum node by issuing the following command: </br> 
+```
+curl -H "Content-Type:application/json" -X POST -d '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}' http://127.0.0.1:8545
+```
+When result is 'false' that means it is now synced ('true' is not synced).
+### Gravity Node
+You can issue the following command to check the sync status of the Gravity Node </br> 
+```
+gravity status 2>&1| jq .SyncInfo.catching_up
+```
+Value of 'false' means that it is now synced, 'true' means that sync is still in process. 
 
-you will need to wait for your Gravity Bridge node and Ethereum Node to fully sync before progressing in the instructions
+### Look at Time Remaining for Gravity Sync
+If you look at your journal logs for gravity-node like so: </br> 
+```
+journalctl -u gravity-node.service -f --output cat
+```
+You should see a message in the logs that looks like (search for 'fast') :  
+```
+9:49PM INF committed state app_hash=76B7FFFA844FA8EABA6E2C400DBE53C22A6F94A36E41922F06C0D57417E118EB height=350595 module=state num_txs=1
+9:49PM INF Fast Sync Rate blocks/s=6.0707074013746825 height=350596 max_peer_height=422352 module=blockchain
+9:49PM INF indexed block height=350595 module=txindex
+```
+You can calculate remaining time in seconds with:
+```
+(max_peer_height - height) / Fast Sync Rate blocks/s
+```
+## Download Blockchain Snapshot 
```

Comment:
this should be marked as optional and note that the highest security way is to sync yourself.

Reply:
Now noted as optional and security note added